fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -823,14 +832,37 @@ ACTOR Future<Void> rocksDBMetricLogger(UID id,
 			cum = stat;
 		}
 
-		for (auto& p : propertyStats) {
+		for (auto& p : intPropertyStats) {
 			auto& [name, property] = p;
 			stat = 0;
 			// GetAggregatedIntProperty gets the aggregated int property from all column families.
 			ASSERT(db->GetAggregatedIntProperty(property, &stat));
 			e.detail(name, stat);
 		}
 
+		std::string propValue;
+		for (auto& p : strPropertyStats) {
+			auto& [name, property] = p;
+			propValue = "";
+			ASSERT(db->GetProperty(cf, property, &propValue));
+			e.detail(name, propValue);
+		}
+
+		rocksdb::ColumnFamilyMetaData cf_meta_data;
+		db->GetColumnFamilyMetaData(cf, &cf_meta_data);
+		int numLevels = static_cast<int>(cf_meta_data.levels.size());
+		std::string levelProp;
+		for (auto& p : levelStrPropertyStats) {
+			auto& [name, property] = p;
+			levelProp = "";
+			for (int level = 0; level < numLevels; level++) {
+				propValue = "";
+				ASSERT(db->GetProperty(cf, property + std::to_string(level), &propValue));
+				levelProp += std::to_string(level) + ":" + propValue + ",";
```

Comment:
Should "," be added to the last one as well?

Reply:
I can add a condition to avoid ",", but I just thought it as an unnecessary check just for logging.

Reply:
it's as simple as `... + (level == numLevels - 1 ? "" : ",")`.